spec/spec_helper.rb

```diff
@@ -3,6 +3,8 @@
 require "ruby-lsp-rspec"
 require "ruby_lsp/internal"
 
+RubyLsp::Addon.load_addons
```

Comment:
I looked at ruby-lsp-rails for inspiration, looks like they load the file directly in tests: 

https://github.com/Shopify/ruby-lsp-rails/blob/4dbe75b67e9f62cd9536ec2394ab2e732cb09316/test/test_helper.rb#L15C1-L15C40

I thought that using `RubyLsp::Addon.load_addons` would be a bit less error-prone since it ensures the path structure is correct. Not sure if there are any downsides to this, like maybe it's considered relying on a private API? Happy to change to `require "ruby_lsp/ruby_lsp_rspec/addon"` if that's the case.